### PR TITLE
Siri intent definition localisation

### PR DESCRIPTION
**Pull request by sharplet**
When I submitted an initial beta build to TestFlight, I got a warning about missing localisations for our supported languages (Italian, French and Polish). This work-in-progress PR first configures the project for Base Internationalisation, then sets up strings files for our various languages. This branch can then become a staging area for the actual translations, which I'll be reaching out to previous contributors for help with.

### CODE REVIEW

File: Sources/Tropos/Extensions/INInteraction.swift [hunk 1]
```diff
@@ -9,7 +9,6 @@ extension INInteraction {
         return INInteraction(intent: intent, response: nil)
     }
 
-    private static let checkWeatherSuggestedPhrase = NSLocalizedString(
-        "Check the weather",
-        comment: "Siri suggestion phrase to check weather")
+    private static let checkWeatherSuggestedPhrase = NSString
+        .deferredLocalizedIntentsString(with: "CheckWeatherSuggestedPhrase") as String
```
Comment (sharplet) [inline Sources/Tropos/Extensions/INInteraction.swift]: This new concept of deferred localisation enables a donated intent to be localised potentially much later, when the device could have changed language.

File: Sources/Tropos/Info.plist [hunk 2]
```diff
@@ -39,6 +39,10 @@
 	</dict>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Tropos does not access your photo library.</string>
+	<key>NSUserActivityTypes</key>
+	<array>
+		<string>CheckWeatherIntent</string>
```
Comment (sharplet) [inline Sources/Tropos/Info.plist]: This was automatically added by Xcode.

File: Sources/TroposIntents/it.lproj/Intents.strings [hunk 3]
```diff
@@ -0,0 +1,6 @@
+"8SKsHj" = "Check weather forecast";
```
Comment (sharplet) [inline Sources/TroposIntents/it.lproj/Intents.strings]: Needs translation into Italian.

File: Sources/TroposIntents/it.lproj/Intents.strings [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+"Hpkcrr" = "${conditionsDescription}";
+
+"RwwC7B" = "Check the current weather forecast";
```
Comment (sharplet) [inline Sources/TroposIntents/it.lproj/Intents.strings]: Needs translation into Italian.

File: Sources/TroposIntents/pl-PL.lproj/Intents.strings [hunk 5]
```diff
@@ -0,0 +1,6 @@
+"8SKsHj" = "Check weather forecast";
```
Comment (sharplet) [inline Sources/TroposIntents/pl-PL.lproj/Intents.strings]: Needs translation into Polish.

File: Sources/TroposIntents/pl-PL.lproj/Intents.strings [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+
+"Hpkcrr" = "${conditionsDescription}";
+
+"RwwC7B" = "Check the current weather forecast";
```
Comment (sharplet) [inline Sources/TroposIntents/pl-PL.lproj/Intents.strings]: Needs translation into Polish.

File: Sources/TroposIntents/sv.lproj/Intents.strings [hunk 7]
```diff
@@ -0,0 +1,6 @@
+"8SKsHj" = "Check weather forecast";
```
Comment (sharplet) [inline Sources/TroposIntents/sv.lproj/Intents.strings]: Needs translation into Swedish.

File: Sources/TroposIntents/sv.lproj/Intents.strings [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+
+"Hpkcrr" = "${conditionsDescription}";
+
+"RwwC7B" = "Check the current weather forecast";
```
Comment (sharplet) [inline Sources/TroposIntents/sv.lproj/Intents.strings]: Needs translation into Swedish.

**Comment by sharplet** [comment]
Hello former Tropos contributors! Would you be interested in contributing translations for Tropos’ new custom Siri shortcut?

- @andreamazz and @ldamon: You previously helped out with our Italian translation in https://github.com/thoughtbot/Tropos/pull/130.
- @tailec, @tomkowz and @sharnik: You previously helped out with our Polish translation in https://github.com/thoughtbot/Tropos/pull/150.
- @aenglund, @mike-burns and @calleerlandsson: You previously helped out with our Swedish translation in https://github.com/thoughtbot/Tropos/pull/153.

If you're able to help out, thanks! The new strings are located at `Sources/TroposIntents/*.lproj/Intents.strings`. Please open a new pull request against this branch, [`intent-localisation`](https://github.com/thoughtbot/Tropos/tree/intent-localisation).

**Comment by sharplet** [comment]
For anyone contributing translations, could you also take a look at `CheckWeatherSuggestedPhrase` in `Sources/Tropos/Resources/*.lproj/Localizable.strings`? This phrase should be a short example phrase that a user might want to say to Siri to check the weather using Tropos, not necessarily a direct translation of "check the weather" if there's something else more idiomatic.

**Comment by sharnik** [comment]
> This is probably terrible, but for now I've added Polish

I've checked and the Polish translations look good. 👍

**Comment by sharplet** [comment]
@sharnik Thank you!

**Comment by sharplet** [comment]
There's something up with the simulators on Circle CI right now. The tests are passing on my machine, but I don't currently have the GM installed, so I'll have to wait before I can verify using the GM (unless someone else wants to have a go).

**Pull request merged**